### DISCUSSION TITLE
Can DRYIOC be used for .Net Framework 4.8 ?

### DISCUSSION THREAD

**Discussion by sfrierdich** [dadhi/DryIoc#585] [General]
Can DRYIOC be used for .Net Framework 4.8 ?   If so is there any sample code showing DRYICO used in .Net Framework 4.8? I believe it can because I think PRISM for .Net Framework used DRYIOC. Also are  there any documentation and tutorials on the web about DRYIOC? If so can you leave me the links

**Comment by dadhi** [comment]
@sfrierdich Yes, you can use it for the .NET 4.8
Regarding the documentation for the DryIoc - you may check the up-to-date version here https://github.com/dadhi/DryIoc/blob/master/docs/DryIoc.Docs/Home.md#users-guide

In regard to the PRISM please check their repository and documentation.
